pom.xml

```diff
@@ -81,10 +81,46 @@
   </build>
 
   <repositories>
+    <repository>
+      <id>openmrs-repo</id>
+      <name>OpenMRS Nexus Repository</name>
+      <url>https://mavenrepo.openmrs.org/public</url>
+    </repository>
+    <repository>
+      <id>openmrs-repo-releases</id>
+      <name>OpenMRS Release Repository</name>
+      <url>https://mavenrepo.openmrs.org/releases</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
+    <repository>
+      <id>openmrs-repo-snapshots</id>
+      <name>OpenMRS Snapshots Repository</name>
+      <url>https://mavenrepo.openmrs.org/snapshots</url>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
```

Comment:
Do we need to specify the snapshots & releases?
Isn't https://mavenrepo.openmrs.org/public enough?

cc @ibacher .